### ISSUE TITLE
Question on safety objects (e.g. Road traffic signs) in MetaDrive

### ISSUE THREAD

**Issue by Aoyama7Hai** [metadriverse/metadrive#684]
Thank you for your solid open source work.

Does metadrive support setting up certain road traffic signs on or around the road, such as signs with a speed limit of 60 mph? Or can this be done by customizing the environment?

**Comment by QuanyiLi** [comment]
Yes. you need to customize the environment on your own. Just like how we create `TrafficBarrier`, you can have your own object defined and then put it into the scene. For the 3D model, sometimes you may want the number to be changed instead of having a specific model for each speed limit. In this case, you can use a shader to achieve this.
